docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "3.2"
+services:
+  wordcamp.test:
+    build:
+      context: .docker
+      dockerfile: Dockerfile
+    volumes:
+      - "./public_html/wp-content:/usr/src/public_html/wp-content"
+      - ".docker/wp-config.php:/usr/src/public_html/wp-config.php"
+    ports:
+      - "80:80"
+      - "443:443"
+      - "1080:1080"
+    command: >
+      bash -c "touch /usr/src/public_html/wp-content/mu-plugins/blocks/build/blocks.min.js &&
+               touch /usr/src/public_html/wp-content/mu-plugins/blocks/build/blocks.min.css &&
```

Comment:
`MIGHT`  - This won't be necessary if we automate the building of this and other things like the Application Tracker

Reply:
Depends upon resolution of https://github.com/WordPress/wordcamp.org/pull/113#discussion_r285101877